_includes/components/who-uses-login.html

```diff
@@ -0,0 +1,15 @@
+  <div class="bg-primary-lightest">
+    <div class="container who-uses-login">
+      <div class="partners list">
+        {{ include.col1 | markdownify }}
+      </div>
+    </div>
+  </div>
+  
+  <div>
```

Comment:
Minor: It existed previously, but from what I can tell, this `<div>` wrapper is redundant and can be safely removed.